core/src/main/java/org/arquillian/smart/testing/SystemPropertyResolver.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.smart.testing;
+
+public class SystemPropertyResolver {
```

Comment:
I'm not sure if wrapping it all up in the Java Bean makes that much sense. I understand that idea here was to encapsulate property names. The challenge I see here is that it will grow quite a bit but also by having one central place will force it to end up in `core` as it will be needed by all modules. Maybe instead of having this class, we can just keep git-related property keys in place and still use good old `System.getProperty`? We are anyway using system properties rule for tests so the argument of improving testability with having this delegate lookup class does not really hold here.